docker/start.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 export PORT=5000
+export START_NGINX=1
```

Comment:
the variable was not exported or initialized and caused also problems during the startup of the container